src/vireshJJun20/Assignment18/MidChar.java

```diff
@@ -0,0 +1,27 @@
+package vireshJJun20.Assignment18;
+
+import javax.swing.plaf.synth.SynthSeparatorUI;
+
+public class MidChar {
+
+	static void findMidChar(String str) { // searching for a middle characters based on mod(even/ odd values) and div values
+		String[] ar=str.split(" ");
+		for (int i= 0; i< ar.length; i++) {
+			String str1= ar[i];
+			int mod= str1.length()%2;
+			int div= str1.length()/2;
+			char mChar;
+			if(mod==0) 
+				mChar= str1.charAt(div-1);
+			else
+				mChar= str1.charAt((div-1)+mod);
```

Comment:
Extra operation + conditions may not be required. Code is working fine but we can write it better like below:
String str = "Harsh Maulik";
	    strArray = str.split(" ");
		System.out.println("Output is below:");
		for (int index=0;index<strArray.length;index++) {
			System.out.println( strArray[index].charAt(((strArray[index].length()-1)/2)) + " is middle character in " + strArray[index]);
		}